### PR TITLE
update readme for resnet50

### PR DESCRIPTION
**Pull request by Ldpe2G**

### CODE REVIEW

**Review by leaves-zwx — APPROVED** [review]

File: resnet50/README.md [hunk 1]
```diff
@@ -1,48 +1,132 @@
...
-### 1. Prepare Traning Data And Pretrain Models
-#### Download Ofrecord
+## Quick Start
+### 1. Prepare Traning Data 
+
+For quick start, you can download the [mini-imagenet](https://oneflow-public.oss-cn-beijing.aliyuncs.com/online_document/dataset/imagenet/mini-imagenet.zip) to train resnet50 on single device. If you want to train resnet50 with the complete imagenet dataset, please see the following instructions.
+
+### 2. Train on single device
+
+Oneflow supports both eager and graph executions.
```
Comment (Flowingsun007) [inline resnet50/README.md:12]:
```suggestion
Oneflow supports execution in eager mode or graph mode.

```

File: resnet50/README.md [hunk 2]
```diff
@@ -1,48 +1,132 @@
...
+									Train batch size of each device.								
+--val-batch-size VAL_BATCH_SIZE
+									Val batch size of each device.		
+--warmup-epochs WARMUP_EPOCHS
+									The number of epoches to do the warmup learning rate scheduling.	
 ```
 
+### 4. Inference on Single Image
+
+#### Download  Pretrain Models for Imagenet
```
Comment (Flowingsun007) [inline resnet50/README.md:115]: Pretrain 是不是改为 >> Pretrained or Pre-trained

File: resnet50/README.md [hunk 3]
```diff
@@ -1,48 +1,132 @@
...
+
+But for know, we have only tested the distributed training on single node with 8 devices. 
+
+#### Train fp32 Model with Distributed Data Prallel
+
+```
+bash examples/train_ddp_fp32.sh
+```
+
+#### Graph Training for fp32
```
Comment (Flowingsun007) [inline resnet50/README.md:42]: fp32 >> FP32

File: resnet50/README.md [hunk 4]
```diff
@@ -1,48 +1,132 @@
...
+
+#### Prepare Ofrecord for The Full Imagenet Dataset
+please refer to: https://github.com/Oneflow-Inc/OneFlow-Benchmark/tree/master/Classification/cnns/tools.
+
+
+Oneflow launches the distributed training the same way as Pytorch. For more details, please refer to the docs: https://oneflow.readthedocs.io/en/master/distributed.html?highlight=distributed#oneflow-distributed
+
+But for know, we have only tested the distributed training on single node with 8 devices. 
+
+#### Train fp32 Model with Distributed Data Prallel
```
Comment (Flowingsun007) [inline resnet50/README.md:36]: fp32 >> FP32

**Review by Flowingsun007 — APPROVED** [review]

**Pull request merged**